k3ng_keyer/k3ng_keyer.ino

```diff
@@ -606,9 +606,38 @@ Recent Update History
   #define tone toneDUE
   #define noTone noToneDUE
 #elif defined(ARDUINO_MAPLE_MINI)
-  #include <SPI.h>
-  #include <Wire.h>
+//  #include <SPI.h>
+//  #include <Wire.h>
   #include <EEPROM.h>
+  void initialize_pins(void);
```

Comment:
Hi, this is the ide from arduino.cc version 1.6.12. The board support is from the STM32duino from: https://github.com/rogerclarkmelbourne/Arduino_STM32
It adds support for the stm32f1×× and other arm family of mcu's ($3 maple mini clone is what I am using now). Without the function declarations you will get these errors: http://pastebin.com/1mu9gtN2
After the function declaration the code compiles. BTW Sorry for sending the pin declaration file, I only wanted to send the changes to the .inorganic.

Maybe moving the function declaration to a stm32duino.h may be better idea?